jinja2/includes/icons/arrows/chevron-down.svg

```diff
@@ -0,0 +1 @@
+<svg class="icon icon-chevron-down" width="26" height="26" xmlns="http://www.w3.org/2000/svg" aria-hidden="true"><g fill-rule="nonzero" fill="none"><circle fill="#222" cx="13" cy="13" r="13"/><path d="M13.544 17.31l6.23-6.16c.301-.297.301-.778 0-1.075l-.726-.719a.776.776 0 0 0-1.087-.001L13 14.237 8.039 9.355a.776.776 0 0 0-1.087.001l-.727.719c-.3.297-.3.778 0 1.075l6.23 6.16c.301.298.789.298 1.09 0z" fill="#FFF"/></g></svg>
```

Comment:
Nit: Missing empty line at end of file

Reply:
These were mine, I'll make the upstream PR